app/views/layouts/board.html.erb

```diff
@@ -13,11 +13,18 @@
   </head>
 
   <body>
-    <div>
-      <p id="notice"><%= notice %></p>
-      <p id="alert"><%= alert %></p>
+    <div style="margin-top: 40px;">
+      <div class="columns">
+        <div class="column is-offset-1 is-2">
+          <%= render 'shared/menu' %>
```

Comment:
Are you sure it is a correct layout to fix? 
Layout called board, not application...